pkg/k8s/apis/cilium.io/v2alpha1/cnc_types.go

```diff
@@ -11,8 +11,8 @@ import (
 //+k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 //+kubebuilder:resource:categories={cilium}
 //+kubebuilder:object:root=true
+//+kubebuilder:deprecatedversion:warning="cilium.io/v2alpha1 CiliumNodeConfig will be deprecated in cilium v1.16; use cilium.io/v2 CiliumNodeConfig"
```

Comment:
wording nit: "is deprecated", not "will be" :-)

Reply:
Does it make sense to keep v2alpha1 non-deprecated for a release? Just to give us time to update docs and procedures?

Reply:
I added a task in the [issue](https://github.com/cilium/cilium/issues/31982) to add the deprecated field for the next release.